examples/source-to-knative-service/app-operator/supply-chain-templates.yaml

```diff
@@ -61,7 +61,7 @@ kind: ClusterSourceTemplate
 metadata:
   name: test
 spec:
-  urlPath: .spec.inputs.source.url
+  urlPath: .status.outputs.url
   revisionPath: .spec.inputs.source.revision
```

Comment:
Should we pick up revision from outputs also?

Reply:
Fixed